cardano-node/src/Cardano/Node/Tracing/Tracers/Resources.hs

```diff
@@ -0,0 +1,40 @@
+{-# LANGUAGE PackageImports #-}
+module Cardano.Node.Tracing.Tracers.Resources
+  (
```

Comment:
unnecessary newline and missing newline between pragma and module nme

Reply:
Done